client/views/admin/users/AddUser.js

```diff
@@ -17,6 +17,8 @@ export function AddUser({ roles, ...props }) {
 	const { value: roleData } = useEndpointData('roles.list', '');
 	const [errors, setErrors] = useState({});
 
+	const { reloadTable, ...otherProps } = props;
```

Comment:
I believe it's safe to do this destructuring at the function parameter declaration, i.e.:

```
export function AddUser({ roles, reloadTable, ...props }) {
```

I don't think `props` requires `reloadTable` property anywhere else.